pipelines/ci/check-pr-author-matches-commit-signature.yml

```diff
@@ -0,0 +1,114 @@
+resources:
+- name: gsp-pr
+  type: pull-request
+  source:
+    access_token: ((pr-build-status))
```

Comment:
pr-build-status should probably be named to indicate sensitivity. pr-build-status-secret or pr-build-status-access-token perhaps

Reply:
`pr-build-status-github-access-token` sounds good. Don't know how to edit the secret in SSM now that it exists, though. :thinking: